src/octopeer-github/main/ElementEventBinding/ScrollIntoViewElementEventBinding.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by Mathias on 2016-05-11.
+ * Class to create ScrollIntoView events
+ */
+class ScrollIntoViewElementEventBinding implements ElementEventBinding {
+    public eventType: string = "scrollintoview";
```

Comment:
This event doesn't exist in standard JavaScript, right? Are we going to trigger this manually?